lib/components/tab.vue

```diff
@@ -36,7 +36,7 @@
         data() {
             return {
                 localActive: this.active && !this.disabled,
-                show: false
+                show: Boolean(this.active && !this.disabled)
```

Comment:
We don't have access to `this.localActive` yet at this stage, as it hasn't been returned to the VM.